tests/integration/conftest.py

```diff
@@ -32,7 +32,7 @@
 from inspirehep.factory import create_app
 
 
-@pytest.yield_fixture(scope='session', autouse=True)
+@pytest.yield_fixture(scope='session')
```

Comment:
Sure, all test I wrote inject `app` directly anyway. Might be worth checking for in-flight PRs.